pkg/datapath/loader/verifier_load_test.go

```diff
@@ -11,13 +11,15 @@ import (
 
 func lxcLoadPermutations() iter.Seq[*config.BPFLXC] {
 	return func(yield func(*config.BPFLXC) bool) {
-		for permutation := range permute(4) {
+		for permutation := range permute(6) {
 			cfg := config.NewBPFLXC(*config.NewNode())
 			cfg.Node.TracingIPOptionType = 1
 			cfg.Node.PolicyDenyResponseEnabled = permutation[0]
 			cfg.AllowIcmpFragNeeded = permutation[1]
 			cfg.EnableIcmpRule = permutation[2]
 			cfg.EnableLRP = permutation[3]
+			cfg.EnableIPv4Fragments = permutation[4]
+			cfg.EnableIPv6Fragments = permutation[5]
```

Comment:
Just to highlight that this doesn't provide test coverage for all the other affected BPF programs. Even if we don't want to permutate there (why not?), we should probably set it to `1` then.